src/components/news-list-item/news-list-item.js

```diff
@@ -1,16 +1,40 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./news-list-item.css";
 
-const NewsListItem = ({ news }) => {
-  const { title, type, url, time } = news;
+import NewsService from "../../services/news-service";
+import Spinner from "../spinner/spinner";
+
+const newsService = new NewsService();
```

Comment:
There's no need in `NewsService` class. 
You can ´explicitly export methods from the service module, like 
```js
export const getNews = async () ...
```